src/Chainweb/Pact/Backend/Utils.hs

```diff
@@ -110,10 +112,10 @@ withSQLiteConnection :: String -> [Pragma] -> Bool -> (SQLiteEnv -> IO c) -> IO
 withSQLiteConnection file ps todelete action = bracket opener closer action
   where
     closer c = do
-      void $ close $ _sConn c
+      void $ close_v2 $ _sConn c
       when todelete (removeFile file)
     opener = do
-      e <- open $ fromString file
+      e <- open_v2 (fromString file) (0x00000002 .|. 0x00000004 .|. 0x00010000) "unix"
```

Comment:
Do you have to pass in the vfs name here, or does `nullPtr` work as default?

Reply:
It seems I have to pass a definitive value here.

Reply:
Is anybody supposed to understand what the opaque bit values mean? Please comment what each is and why we should care.

Reply:
From the sqlite docs:

> `If the fourth parameter is a NULL pointer then the default sqlite3_vfs object is used.`

So you definitely want to prefer `nullPtr` to `"unix"` here